### ISSUE TITLE
Store multiple windows in session

### ISSUE THREAD

**Issue by christian-rauch** [gnunn1/tilix#1744]
At the moment, tilix (1.7.7) can only store a single session in a window to file. It would be useful if a session could consist of multiple windows which could be stored to file and restored. This is especially useful for multi-monitor setups.
The settings contain an option for "Save and restore window state", but this only applies to a single window.

For inspiration, Terminator implements such a feature.

**Comment by gaui** [comment]
I now create a seperate file for each session, because this is not a possible. But yeah, it would be awesome to be able to save an instance in a single file (multiple sessions).

**Comment by bflorat** [comment]
I agree. This is a very important feature. Int he mean time, here's a workaround using a wrapper shell :


```
#!/bin/bash

TILIX_SESSIONS_FOLDER="~/tilix-sessions"

TILIX_OPTS=""

for session in $TILIX_SESSIONS_FOLDER/*; do
   TILIX_OPTS="$TILIX_OPTS -s $session"
done

tilix $TILIX_OPTS

```
